Jenkinsfile.cloud

```diff
@@ -224,12 +224,24 @@ node(NODE) {
         }
     }
 
-    stage("Sync Out") {
+    par_stages["rsync-out"] = { -> stage("rsync out") {
         withCredentials([
             string(credentialsId: params.ARTIFACT_SERVER, variable: 'ARTIFACT_SERVER'),
             sshUserPrivateKey(credentialsId: params.ARTIFACT_SSH_CREDS_ID, keyFileVariable: 'KEY_FILE'),
         ]) {
             utils.rsync_dir_out(ARTIFACT_SERVER, KEY_FILE, images)
         }
-    }
+    } }
+    par_stages("s3-out") = -> { stage("s3 upload") {
```

Comment:
Should this be `par_stages["s3-out"]`?

Reply:
It's not required but yeah, let's keep them in sync.  (It's sort of annoying that `parallel` wants a map and not just an array/set), and we have to just make up a key.